### ISSUE TITLE
Generate and allow writing output after every chapter?

### ISSUE THREAD

**Issue by gregschwartz** [skoji/gepub#136]
Very much appreciating this gem, it's been a huge help! I'm pulling down serially posted stories to put them on to my Kindle. (Mostly from the amazing [/r/HFY](https://www.reddit.com/r/HFY) but also other places.)

For example, it retrieves [The Sacred Noise part 1](https://www.reddit.com/r/HFY/comments/mqk583/the_sacred_noise_oc/) and [part 2](https://www.reddit.com/r/HFY/comments/na860t/the_sacred_noise_ii_oc/). 

Then it moves on to the next series, grabbing [Large Scale Engineering](https://www.reddit.com/r/HFY/comments/mvji51/large_scale_engineering/) and then [Economies of Scale](https://www.reddit.com/r/HFY/comments/n0j1f4/economies_of_scale/) (which is part 2).

Repeat for all the series I'm reading.

I have it working to retrieve everything _within_ the `book.ordered do` loop, then exporting everything once after finishing retrieving all series. 

However, I'd like to export the ePub in between each series, in case retrieving a series causes a crash. (These examples are 2 parts, some stories are 50+ parts.) So I tried this after _each_ series:

```
epubname = File.join(File.dirname(__FILE__), "Stories " + dateForFilename + ".epub")
book.generate_epub(epubname)
```

However the nav.xhtml is not regenerated after the first `generate_epub()` call. So it only contains the first series. And the resulting ePub has some other oddities, too.

tl;dr: is there a way to build the book without a `book.ordered do` loop? Or make it correctly regenerate if you call `generate_epub()` multiple times?

Thank you!
